frosh/devenv-meta/6.4/root/devenv.nix

```
@@ -0,0 +1,128 @@
+{ pkgs, config, ... }:
+
+{
+  enterShell = ''
+    rm -f .devenv/bin
+    ln -sf ${pkgs.buildEnv { name = "devenv"; paths = config.packages; ignoreCollisions = true; }}/bin .devenv/bin
```

```suggestion
    ln -sf ${pkgs.buildEnv { name = "devenv"; paths = config.packages; ignoreCollisions = true; }}/bin .devenv/profile
```

It will be soon renamed to `.devenv/profile`

https://github.com/cachix/devenv/issues/167#issuecomment-1363900892

We should adopt it for now and remove that code then